client/layout/store-alerts/index.js

```diff
@@ -62,6 +63,109 @@ class StoreAlerts extends Component {
 		}
 	}
 
+	renderActions( alert ) {
+		const { updateNote } = this.props;
+		const actions = alert.actions.map( action => {
+			const markStatus = () => {
+				updateNote( alert.id, { status: action.status } );
+			};
+			return (
+				<Button
+					key={ action.name }
+					isDefault
+					href={ action.url }
+					onClick={ '' === action.status ? noop : markStatus }
+				>
+					{ action.label }
+				</Button>
+			);
+		} );
+
+		// TODO: should "next X" be the start, or exactly 1X from the current date?
+		const snoozeOptions = [
+			{
+				newDate: moment()
+					.add( 4, 'hours' )
+					.unix(),
+				label: __( 'Later Today', 'woocommerce-admin' ),
```

Comment:
Yeah maybe this should just say, "In a few more hours" since it could technically be the next day right?

Reply:
As implemented yes it might be the next day. We could alternatively make it say, 6pm of the current day and hide the option if it's after 6?